pyuvdata/uvdata/uvfits.py

```diff
@@ -954,6 +937,15 @@ def write_uvfits(
                 fix_autos=fix_autos,
             )
 
+        if spoof_nonessential:
+            warnings.warn(
+                (
+                    "The spoof_nonessential parameter is deprecated and will be "
+                    "removed in a future release."
```

Comment:
Maybe add something to let users know that all the uvfits metadata are now properly defined and don't need to be spoofed?

Reply:
Done in 587adde5.